src/platform/forms/save-in-progress/SaveInProgressDevModal.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import ErrorableTextArea from '@department-of-veterans-affairs/formation-react/ErrorableTextArea';
+import ErrorableSelect from '@department-of-veterans-affairs/formation-react/ErrorableSelect';
+
+import environment from 'platform/utilities/environment';
+
+const SipsDevModal = props => {
+  const [isModalVisible, toggleModal] = useState(false);
+  const [sipsData, setSipsData] = useState(null);
+  const [sipsUrl, setSipsUrl] = useState(null);
+  const [errorMessage, setError] = useState('');
+
+  const availablePaths = props?.route?.pageList.map(page => page.path) || [];
+
+  const openSipsModal = () => {
+    setSipsData(JSON.stringify(props.form.data, null, 2));
+    setSipsUrl(props.locationPathname);
+    toggleModal(true);
+  };
+
+  const saveData = (event, type) => {
+    event.preventDefault();
+    const { formId, version, data } = props.form;
+    const parsedData = JSON.parse(sipsData);
+    const newData =
+      type === 'merge' ? Object.assign({}, data, parsedData) : sipsData;
+    setError('');
+    props.saveAndRedirectToReturnUrl(formId, newData, version, sipsUrl);
+    toggleModal(false);
+  };
+
+  const handleChange = value => {
+    try {
+      JSON.parse(value);
+      setError('');
+    } catch (err) {
+      setError(err?.message || err?.name);
+    }
+    setSipsData(value);
+  };
+
+  return environment.isProduction() ? null : (
```

Comment:
You could possibly move this to the end of the file like `export default (environment.isProduction()  ? () => null : SipsDevModal)`. It could be clearer as a non-prod thing that way...but also i wonder if this would be helpful at all in prod behind a secret key in localStorage or something. What do you think?

Reply:
Yeah, I can do that!

Funny, I was just talking to Dan about this PR. I think I need to change the color of the link, but then it is hard to notice it. Maybe I should change it to a gear icon?

Reply:
I'm not sure if it'd be useful in production. I can't think of a use case... I mean, I've been on calls with veteran's having problems, and we can usually talk them through getting their SIPs data from the dev tools network. I think it'd be difficult to get something like this approved for use in production either way.